src/Traits/LoadsFilesTrait.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Michaels\Manager\Traits;
+
+/**
+ * Allows a class to have file loading capabilities.
+ *
+ * @implements Michaels\Manager\Contracts\ManagesItemsInterface
+ * @package Michaels\Manager
+ */
+
+use Michaels\Manager\Contracts\DecoderInterface;
+use Michaels\Manager\FileLoader;
+
+trait LoadsFilesTrait
+{
```

Comment:
Actually, for now, you can drop that thought. I didn't notice that you actually add the manager as a dependency in `LoadsFilesTrait`. I like that approach, but it is unique in the system, so I'm not sure if it would be better to leave it or to rework it a it so that it just manipulates the current `ManagesItemsTrait` directly, simply for consistency in the library. I'll stew over that and probably decide after the merge. Right now, if it ain't broke, don't fix it! lol